larq/metrics_test.py

```diff
@@ -15,54 +16,48 @@ def test_scope():
 
 
 def test_config():
-    mcv = metrics.FlipRatio(
-        values_shape=[3, 3], values_dtype="int16", name="mcv", dtype=tf.float16
-    )
+    mcv = metrics.FlipRatio(values_dtype="int16", name="mcv", dtype=tf.float16)
     assert mcv.name == "mcv"
     assert mcv.stateful
     assert mcv.dtype == tf.float16
     assert mcv.values_dtype == tf.int16
-    assert mcv.values_shape == [3, 3]
+    # assert mcv.values_shape == [3, 3]
 
     mcv2 = metrics.FlipRatio.from_config(mcv.get_config())
     assert mcv2.name == "mcv"
     assert mcv2.stateful
     assert mcv2.dtype == tf.float16
     assert mcv2.values_dtype == tf.int16
-    assert mcv2.values_shape == [3, 3]
+    # assert mcv2.values_shape == [3, 3]
 
 
 def test_metric(eager_mode):
-    mcv = metrics.FlipRatio([2])
+    mcv = metrics.FlipRatio()
 
-    assert 0 == mcv.result().numpy()
-    assert 0 == mcv.total.numpy()
-    assert 0 == mcv.count.numpy()
```

Comment:
I think we should actually keep those tests and just manually call `mcv.build` before.